### PR TITLE
feat: LEAP-572: Implement interface indicator for expired plans

### PR DESCRIPTION
**Pull request by juliosgarbi**
### PR fulfills these requirements
- [x] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [x] Frontend

### Describe the reason for change
Tooltips or prompts should be implemented to provide explanations when users hover over or attempt to click these greyed-out buttons.

#### What is the new behavior?
Users will not be able to import additional data or create new projects, or label any data, when user tries to click on this buttons, it's showing a tooltip explaining to the user why it's not possible

Screenshot 2024-02-27 at 12 04 24

#### What feature flags were used to cover this change?
fflag_feat_front_leap_482_self_serve_short

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [ ] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [ ] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
_(for bug fixes/features, be as precise as possible. ex. Authentication, Annotation History, Review Stream etc.)_

### CODE REVIEW

File: src/components/Common/SDKButtons.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const SDKButton = ({ eventName, ...props }) => {
       onClick={() => {
         sdk.invoke(eventName);
       }}
-    />
+    />
```
Comment (yyassi-heartex) [inline src/components/Common/SDKButtons.js:14]:
```suggestion
    />
```

**Review by yyassi-heartex — APPROVED** [review]
approved, just one minor replacement

File: src/components/DataManager/Toolbar/instruments.js [hunk 2]
```diff
@@ -13,20 +13,31 @@ import { LoadingPossum } from "./LoadingPossum";
 import { OrderButton } from "./OrderButton";
 import { RefreshButton } from "./RefreshButton";
 import { ViewToggle } from "./ViewToggle";
+import { Tooltip } from "../../Common/Tooltip/Tooltip";
+import { Block } from "../../../utils/bem";
```
Comment (hlomzik) [inline src/components/DataManager/Toolbar/instruments.js:17]: please move them up to fit the order

Comment (juliosgarbi) [reply src/components/DataManager/Toolbar/instruments.js:17]: Alphabetical order?

File: src/components/DataManager/Toolbar/instruments.js [hunk 3]
```diff
@@ -14,19 +16,28 @@ import { OrderButton } from "./OrderButton";
...
+  minWidth: '110px',
+  justifyContent: 'space-between',
 };
 
+// Check if user is self-serve
+const isSelfServe = isFF(FF_SELF_SERVE) && window.APP_SETTINGS.billing.enterprise;
+// Check if user is self-serve and has expired trial
+const isSelfServeExpiredTrial = isSelfServe && window.APP_SETTINGS.billing.checks.is_license_expired && !window.APP_SETTINGS.subscription.current_period_end;
+// Check if user is self-serve and has expired subscription
+const isSelfServeExpiredSubscription = isSelfServe && window.APP_SETTINGS.subscription.current_period_end && new Date(window.APP_SETTINGS.subscription.current_period_end) < new Date();
```
Comment (hlomzik) [inline src/components/DataManager/Toolbar/instruments.js:29]: move these long items to consts as well as in LSP

**Review by hlomzik — CHANGES_REQUESTED** [review]

**Pull request merged**